clojure_liberator/src/clojure_liberator/data_access.clj

```diff
@@ -0,0 +1,22 @@
+(ns clojure-liberator.data-access)
+(use 'korma.db)
+(use 'korma.core)
+(require '[clojure.string :as str])
```

Comment:
I don't see any other references to `str` in this file. Is this line doing something magic?